docs/edge.rst

```diff
@@ -217,10 +217,12 @@ Configure the cluster
 Configuration
 '''''''''''''
 
-Next, go through the cluster configuration process. On the left-hand side, you
-can choose a subscription plan. As you select each subscription plan, you will
-see that the values for CPU, RAM, and storage per node change accordingly in
-the middle panel that shows the node specification.
+Next, go through the cluster configuration process. You will see your custom
+Edge region is selected, so no region selection is necessary. You can move
+directly to the cluster configuration. On the left-hand side, you can choose a
```

Comment:
Not sure if this is accurate or maybe I misread. When an edge region is selected, the plan selector is not show and instead you configure the cluster manually.
Edge region selected 👇🏼 
![Screenshot 2021-08-20 at 17 45 47](https://user-images.githubusercontent.com/855218/130266685-79fe8add-07fe-464b-8c12-0fb98361627f.png)